openfisca_us/variables/expense/phone_expense.py

```diff
@@ -0,0 +1,12 @@
+from openfisca_us.model_api import *
+
+
+class telephone_expense(Variable):
```

Comment:
Not `phone`?

Reply:
I think telephone is more specific to the phone line and not mobile.

Reply:
USDA allows deducting mobile expenses too: https://www.fns.usda.gov/snap/eligibility/resources/non-land-line-telephone-expenses-allowances-deductions

This variable also applies to Lifeline, which is for mobile, so if we need landline specifically it should be a separate variable.

Reply:
Ah, OK. Reverted, thanks